internals/getPermissionHeaders.js

```diff
@@ -53,4 +53,42 @@ const getCsp = ({ isExtension } = { isExtension: false }) =>
     .join(' ')
     .replace(/ ;/g, ';')
 
-module.exports = { getCsp }
+// Generated with https://www.permissionspolicy.com/
+const getPermissionsPolicy = () =>
+  `
+    accelerometer=("https://api.sardine.ai"),
+    ambient-light-sensor=(),
+    autoplay=(),
+    bluetooth=(self),
+    camera=("https://global.transak.com" "https://global-stg.transak.com"),
+    cross-origin-isolated=(),
+    display-capture=(),
+    document-domain=(),
+    encrypted-media=(),
+    execution-while-not-rendered=(),
+    execution-while-out-of-viewport=(),
+    fullscreen=(self "https://global.transak.com" "https://global-stg.transak.com"),
+    geolocation=(),
+    gyroscope=("https://api.sardine.ai"),
```

Comment:
This is annoying for maintenance: Sardine is a dependency of Transak. We won't know when Transak changes their dependencies, or if Sardine started requiring geolocation.

Should we loosen accelerometer,ambient-light-sensor,camera,encrypted-media,geolocation,gyroscope,magnetometer,microphone,publickey-credentials-get,web-share to `=*`?

Reply:
Although their docs say to put `allow="camera;microphone;fullscreen;payment"` on iframe, so it's already further restricted :shrug: